### PR TITLE
Created react app + deployment to Heroku

### PR DESCRIPTION
**Pull request by gurleen-sanghera**
Changed the package.json file

### CODE REVIEW

File: UI/aoc-app/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "name": "aoc-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-scripts": "3.4.0"
```
Comment (kvptkr) [inline UI/aoc-app/package.json:11]: Why did you have to change this from its initial value of `3.4.4`?

**Review by kvptkr — CHANGES_REQUESTED** [review]
Looks good for the most part, could you just add a markdown file with some instructions on how to push to Heroku and where we can see the live demo?

File: UI/aoc-app/package.json (hunk 1 above)
Comment (gurleen-sanghera) [reply UI/aoc-app/package.json:11]: The newest react version wasn't compatible with the Heroku buildpack.

**Comment by kvptkr** [comment]
README updated here: https://github.com/AcademyofCleaning/AcademyofCleaning/pull/5

minor nit: remove the .DS_Store file in the root folder and add it to the .gitignore

LGTM for the most part though, make sure to squash your commits before merging! Also, see my comments in the other PR for how to integrate that change!

**Review by kvptkr — APPROVED** [review]

**Review by matthewfollegot — APPROVED** [review]
Looks good to be merged, just a few nitpicky things before doing so (Kanav already mentioned the second point)

1. Can we change the PR name to be more reflective of the work that was done (i.e. story name)?
1. #5 should be integrated into this PR - the AC of this PR is the Heroku deployment so any resources supporting that should be in this PR

**Pull request merged**